src/ssl.c

```diff
@@ -23396,36 +23580,9 @@ int wolfSSL_sk_CIPHER_description(WOLFSSL_CIPHER* cipher)
     pv.major = cipher_names[offset].major;
     pv.minor = cipher_names[offset].minor;
     protocol = wolfSSL_internal_get_version(&pv);
+    
+    name = GetCipherSegment(cipher, n);
```

Comment:
Should check return value of GetCipherSegment()

Reply:
Added checking return value, thanks